### PR TITLE
Fix install, upgrade, deploy, and manage issues

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

This PR is to fix the following issues:
- https://github.com/pingcap/docs-cn/issues/8048
- https://github.com/pingcap/docs/issues/6809
- https://github.com/pingcap/docs/issues/6536
- https://github.com/pingcap/docs/issues/6537
- https://github.com/pingcap/docs/issues/6535 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: faq/manage-cluster-faq.md [hunk 1]
```diff
@@ -43,16 +45,20 @@ mysql -h 127.0.0.1 -uroot -P4000
...
+- 未使用 load balancer：执行 kill 命令，TiDB 的组件会做 graceful 的 shutdown。
 
 ### TiDB 里面可以执行 kill 命令吗？
 
-- 可以 kill DML 语句，首先使用 `show processlist`，找到对应 session 的 id，然后执行 `kill tidb [session id]`。
-- 可以 kill DDL 语句，首先使用 `admin show ddl jobs`，查找需要 kill 的 DDL job ID，然后执行 `admin cancel ddl jobs 'job_id' [, 'job_id'] ...`。具体可以参考 [admin 操作](/sql-statements/sql-statement-admin.md)。
+[Kill](/sql-statements/sql-statement-kill.md) 命令默认禁用，如果要执行 Kill 命令，需要先设置 `compatible-kill-query = true` 或者开启 Global Kill 特性 （实验特性）。
+
+- Kill DML 语句：执行 `show processlist`，找到对应的 session id，然后执行 `kill tidb [session id]`。
+- Kill DDL 语句：执行 `admin show ddl jobs`，查找需要 kill 的 DDL job ID，然后执行 `admin cancel ddl jobs 'job_id' [, 'job_id'] ...`。具体可以参考 [admin 操作](/sql-statements/sql-statement-admin.md)。
```
Comment (xuyifangreeneyes) [inline faq/manage-cluster-faq.md:57]: - Kill DML 语句：查询 `information_schema.cluster_processlist` 得到正在执行 DML 语句的 TiDB 实例地址和 session ID，然后执行 kill 语句。TiDB 从 v6.1.0 起新增 Global Kill 功能（由 [`enable-global-kill`](/tidb-configuration-file.md#enable-global-kill-从-v610-版本开始引入) 配置项控制，默认启用）。启用 Global Kill 功能时，直接执行 `kill session_id` 即可。对于 TiDB v6.1.0 之前的版本，或未启用 Global Kill 功能时，`kill session_id` 默认不生效，客户端需要连接到正在执行 DML 语句的 TiDB 实例，然后执行 `kill tidb session_id` 才能 kill DML 语句。如果客户端连接到其他 TiDB 实例或者客户端和 TiDB 集权之间有代理，`kill tidb session_id` 可能会被路由到其他的 TiDB 实例，从而错误地终止其他会话。具体可以参考请 [`KILL`](/sql-statements/sql-statement-kill.md)。
- Kill DDL 语句：执行 `admin show ddl jobs`，查找需要 kill 的 DDL job ID，然后执行 `admin cancel ddl jobs 'job_id' [, 'job_id'] ...`。具体可以参考 [`ADMIN`](/sql-statements/sql-statement-admin.md)。

File: faq/manage-cluster-faq.md [hunk 2]
```diff
@@ -43,16 +45,25 @@ mysql -h 127.0.0.1 -uroot -P4000
...
 
-- 可以 kill DML 语句，首先使用 `show processlist`，找到对应 session 的 id，然后执行 `kill tidb [session id]`。
-- 可以 kill DDL 语句，首先使用 `admin show ddl jobs`，查找需要 kill 的 DDL job ID，然后执行 `admin cancel ddl jobs 'job_id' [, 'job_id'] ...`。具体可以参考 [admin 操作](/sql-statements/sql-statement-admin.md)。
+- Kill DML 语句：
+
+    查询 `information_schema.cluster_processlist`，得到正在执行 DML 语句的 TiDB 实例地址和 session ID，然后执行 kill 语句。
+
+    TiDB 从 v6.1.0 起新增 Global Kill 功能（由 [`enable-global-kill`](/tidb-configuration-file.md#enable-global-kill-从-v610-版本开始引入) 配置项控制，默认启用）。启用 Global Kill 功能时，直接执行 `kill session_id` 即可。
+
+    对于 TiDB v6.1.0 之前的版本，或未启用 Global Kill 功能时，`kill session_id` 默认不生效，客户端需要连接到正在执行 DML 语句的 TiDB 实例，然后执行 `kill tidb session_id` 才能 kill DML 语句。如果客户端连接到其他 TiDB 实例或者客户端和 TiDB 集权之间有代理，`kill tidb session_id` 可能会被路由到其他的 TiDB 实例，从而错误地终止其他会话。具体可以参考请 [`KILL`](/sql-statements/sql-statement-kill.md)。
```
Comment (shichun-0415) [inline faq/manage-cluster-faq.md:60]:
```suggestion
    对于 TiDB v6.1.0 之前的版本，或未启用 Global Kill 功能时，`kill session_id` 默认不生效，客户端需要连接到正在执行 DML 语句的 TiDB 实例，然后执行 `kill tidb session_id` 才能 kill DML 语句。如果客户端连接到其他 TiDB 实例或者客户端和 TiDB 集群之间有代理，`kill tidb session_id` 可能会被路由到其他的 TiDB 实例，从而错误地终止其他会话。具体可以参考 [`KILL`](/sql-statements/sql-statement-kill.md)。
```

Comment (shichun-0415) [inline faq/manage-cluster-faq.md:60]:
```suggestion
    对于 TiDB v6.1.0 之前的版本，或未启用 Global Kill 功能时，`kill session_id` 默认不生效，客户端需要连接到正在执行 DML 语句的 TiDB 实例，然后执行 `kill tidb session_id` 才能 kill DML 语句。如果客户端连接到其他 TiDB 实例或者客户端和 TiDB 集群之间有代理，`kill tidb session_id` 可能会被路由到其他的 TiDB 实例，从而错误地终止其他会话。具体可以参考请 [`KILL`](/sql-statements/sql-statement-kill.md)。
```

**Comment by easonn7** [comment]
LGTM

File: faq/manage-cluster-faq.md [hunk 3]
```diff
@@ -43,16 +45,25 @@ mysql -h 127.0.0.1 -uroot -P4000
 
 ### 如何规范停止 TiDB？
 
-可以直接 kill 掉所有服务。如果使用 kill 命令，TiDB 的组件会做 graceful 的 shutdown。
+- 使用了 load balancer （推荐）：先停止 load balancer，然后执行 `SHUTDOWN` 语句。此时 TiDB 会根据 [graceful-wait-before-shutdown](/tidb-configuration-file.md#graceful-wait-before-shutdown-从-v50-版本开始引入) 设置值等所有会话断开，然后停止运行。
```
Comment (TomShawn) [inline faq/manage-cluster-faq.md:48]:
```suggestion
- 若使用了 load balancer（推荐）：先停止 load balancer，然后执行 `SHUTDOWN` 语句。此时 TiDB 会根据 [`graceful-wait-before-shutdown`](/tidb-configuration-file.md#graceful-wait-before-shutdown-从-v50-版本开始引入) 设置值等待所有会话断开，然后停机。
```

File: faq/manage-cluster-faq.md [hunk 4]
```diff
@@ -43,16 +45,25 @@ mysql -h 127.0.0.1 -uroot -P4000
...
+
+- 未使用 load balancer：执行 `SHUTDOWN` 语句，TiDB 组件会做 graceful shutdown。
```
Comment (TomShawn) [inline faq/manage-cluster-faq.md:50]:
```suggestion
- 若未使用 load balancer：执行 `SHUTDOWN` 语句，TiDB 组件会做 graceful shutdown。
```

File: faq/manage-cluster-faq.md [hunk 5]
```diff
@@ -43,16 +45,25 @@ mysql -h 127.0.0.1 -uroot -P4000
...
 
 ### TiDB 里面可以执行 kill 命令吗？
 
-- 可以 kill DML 语句，首先使用 `show processlist`，找到对应 session 的 id，然后执行 `kill tidb [session id]`。
-- 可以 kill DDL 语句，首先使用 `admin show ddl jobs`，查找需要 kill 的 DDL job ID，然后执行 `admin cancel ddl jobs 'job_id' [, 'job_id'] ...`。具体可以参考 [admin 操作](/sql-statements/sql-statement-admin.md)。
+- Kill DML 语句：
+
+    查询 `information_schema.cluster_processlist`，得到正在执行 DML 语句的 TiDB 实例地址和 session ID，然后执行 kill 语句。
```
Comment (TomShawn) [inline faq/manage-cluster-faq.md:56]:
```suggestion
    查询 `information_schema.cluster_processlist`，获取正在执行 DML 语句的 TiDB 实例地址和 session ID，然后执行 kill 命令。
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

**Comment by tier-cap** [comment]
LGTM

**Pull request merged**